content/rc/billing-and-payments.md

```diff
@@ -72,13 +72,11 @@ The following tabs are available:
 
 To download an invoice:
 
-1. Sign in to the Redis Cloud [admin portal](https://app.redislabs.com/).  (Create an account if you don't already have one.)
+1. In the Redis Cloud [admin portal](https://app.redislabs.com/), select **Billing & Payments** and then make sure the Billing History tab is selected.
```

Comment:
```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments > Billing History**.
```

Or you could remove the "Billing History" part and just specify this tab in the next step:

```suggestion
1. In the Redis Cloud [admin console](https://app.redislabs.com/), select **Billing & Payments**.
```